bin/jsindex

```diff
@@ -51,12 +51,12 @@ if (empty($src)) {
 
 $indexer = new \cebe\jssearch\Indexer();
 
-$files = [];
 foreach($src as $dir) {
-	$files = array_merge($files, findFiles($dir));
+	echo "Processing $dir\n";
+	$files = findFiles($dir);
 
 	if (empty($files)) {
-		error("No files where found in $dir.");
+		echo("No files where found in $dir.");
 	}
 
 	$indexer->indexFiles($files, $dir);
```

Comment:
this should not call `indexFiles()` if $files is empty, so an else should be added here.